pkg/kube/inject/testdata/inject/app_probe/hello-probes.yaml

```diff
@@ -1,43 +0,0 @@
-apiVersion: apps/v1
```

Comment:
All of app_probe/ was moved to inject/ to share the same logic. This one shows as deleted as its a duplicate